run_migrations.py

```
@@ -57,20 +58,64 @@ def is_first_time():
+        # Try to get table names - this will fail if no tables exist or DB is not initialized
+        tables = await connection.execute_query(
+            "SELECT name FROM aerich WHERE type='table';"
+        )
+
+        # Filter out sqlite internal tables
+        user_tables = [
+            table[0] for table in tables if not table[0].startswith("aerich")
+        ]
+
+        return len(user_tables) > 0
+    except (OperationalError, AttributeError, Exception) as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)